core/arch/arm/include/tee/entry.h

```diff
@@ -25,8 +25,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef TEE_ENTRY_H
-#define TEE_ENTRY_H
+#ifndef ENTRY_H
```

Comment:
I thought the convention was to use the file name only. Actually many files in OP-TEE do not follow this recommendation, but I see how it is better. Will fix.